docs/source/getting-started/command-line-interface/run.rst

```diff
@@ -63,3 +63,21 @@ To start the backend server and skip the process cleanup::
 
   # successful output looks the same as without the flag
 
+To enable log parsing for errors, set ``ELASTIC_SEARCH_PATH`` and ``LOGSTASH_PATH`` variables to point to elasticsearch and logstash binaries. For example:
+
+  $ export ELASTIC_SEARCH_PATH=<path_to_elastic_search> #last tested with 7.8.0_2
+  $ export LOGSTASH_PATH=<path_to_logstash> #last tested with 6.8.10; versions 7+ have an unresolved issue with parsing breaking the Augur use-case
+  $ export AUGUR_HOME=<path_to_augur>
+
+Start the http server with:
+
+  $ cd $AUGUR_HOME/log_analysis/http
+  $ python http_server.py
+
+Then run augur with ``logstash`` flag:
+  $ augur run --logstash
+
+If you'd like to clean all previously collected errors, run:
+  $ augur run --logstash-with-cleanup
```

Comment:
Please wrap all code examples in a `.. code-block:: bash` directive so it's formatted properly. You could also shortcut this by using 2 colons at the end of the line, like:
```
Then run augur with ``logstash`` flag::
  $ augur run --logstash
```
This will format the lines below as a code block. There are lots of other examples of both styles throughout the docs if you need a reference.